op-e2e/helper.go

```diff
@@ -1,30 +1,16 @@
 package op_e2e
 
 import (
-	"flag"
 	"os"
 	"testing"
 
 	"github.com/ethereum/go-ethereum/log"
 )
 
-var enableParallelTesting bool = true
-
-// Init testing to enable test flags
```

Comment:
This code has moved around a bit since I've been following the e2e tests.

I'm not really sure why we've opted to move it out of a `_test.go` file and into `helper.go`.  It has the very unfortunate side effect of forcing anyone who includes this package to get all of the test flags added to their execution environment, which strikes me as rather odd.  But, odd things are often done for a purpose, so hopefully someone can let me know if/how this change breaks others.

One alternative, which I considered but didn't love, would be to take the external data structures and helper functions that the shims are also consuming, and put them into their own package.  That way the shims could get access to them without calling this init code, but if we don't need to, it seems better not to.

Reply:
Looks like this code has been pulled into `op-e2e/config/config.go`.  So, I've gone ahead and done the alternative discussed above, of moving the common structures into their own package.

FWIW, I think it's a bad idea to call `testing.Init()` and parsing flags in `func init()`, because it ensures anyone who wants to link to your package ends up having their flags broken.  I don't quite understand the big picture as to why these changes are being done, but if anyone agrees, I'm happy to submit a separate PR moving the flag parsing out of the package initialization.

Reply:
The init behaviour is pretty foreign to me (I'm really a Java dev pretending to be a go dev...) - it's been shoved around a bit as part of trying to separate and simplify the test code and then have implementation details in helper functions.  There wasn't really an intention for the e2e package to be needed to be imported by anyone else to be honest because it's really all just tests. There's some movement towards the helpers being in separate modules which I do think make sense so I think moving the common structures to their own module makes sense too.
I'm not really sure where the init code should live but I don't think it's in the current location by design, more just that's where it wound up because it worked at the time.